include/boost/url/params.hpp

```diff
@@ -217,43 +187,21 @@ class params
     assign(FwdIt first, FwdIt last);
 
 private:
-    /** Assignment from forward iterators
-
-        Assign @ref params from a range of @ref params::value_type.
-
-        Each instance of @ref params::value_type is a view of a query <key, value> pair.
-
-        @tparam FwdIt Forward Iterator Type
-
-        @param first An iterator to the beginning of the query parameter range
-
-        @param last An iterator to the end of the query parameter range
-
-     */
     template<class FwdIt>
     void
     assign(FwdIt first, FwdIt last,
         std::forward_iterator_tag);
 
 public:
 #ifndef BOOST_URL_DOCS
-    /** Assignment from input iterators is deleted
-
-        @note <!-- Doxygen cannot render ` = delete` -->
-     */
+    // Doxygen cannot render ` = delete`
     template<class FwdIt>
     void
     assign(FwdIt first, FwdIt last,
         std::input_iterator_tag) = delete;
 #endif
 
-    /** <!-- @} --> */
-
-    /** <!-- @name Element Access -->
-        <!-- @{ -->
-     */
-
-    /** Access element at specified position with bounds checking
+    /** Access specified element with bounds checking
```

Comment:
Functions which return values almost always have briefs that start with the word "Return"

"Return the indexed element, with bounds checking."